modules/nextflow/src/main/groovy/nextflow/processor/PublishDir.groovy

```diff
@@ -483,6 +483,9 @@ class PublishDir {
         catch( FileAlreadyExistsException e ) {
             // ignore
         }
+        catch( Exception e ) {
+            log.warn "Failed to create directory for publishing file: ${dir.toUriString()}"
```

Comment:
But this changes the existing behavior which was causing the execution to stop, instead not only a warning is returned. Is that made on purpose?